client/blocks/color-scheme-picker/index.jsx

```diff
@@ -0,0 +1,68 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import QueryPreferences from 'components/data/query-preferences';
+import { savePreference, setPreference } from 'state/preferences/actions';
+import { getPreference } from 'state/preferences/selectors';
+import { ColorSchemes } from './constants';
+import FormRadiosBar from 'components/forms/form-radios-bar';
+
+class ColorSchemePicker extends PureComponent {
+	static propTypes = {
+		temporarySelection: PropTypes.bool,
+		onSelection: PropTypes.func,
+		// Connected props
+		colorSchemePreference: PropTypes.string,
+		saveColorSchemePreference: PropTypes.func,
+	};
+
+	handleColorSchemeSelection = event => {
+		const { temporarySelection, onSelection, saveColorSchemePreference } = this.props;
+		const addSelection = handleFunction => handleFunction( event.currentTarget.value );
+		temporarySelection && addSelection( onSelection );
+		addSelection( saveColorSchemePreference );
+	};
```

Comment:
`addSelection` seems like an unnecessary abstraction. This would IMO read better and avoid an extra function instantiation if done more or less like:

```js
const { value } = event.currentTarget;
if ( temporarySelection ) {
  onSelection( value );
}
saveColorSchemePreference( value );
```